### PR TITLE
Add multiple bundle options for Kubeflow

### PR DESCRIPTION
**Pull request by knkski**
Adds the ability to set `KUBEFLOW_BUNDLE` to one of `full`, `lite`, or `edge`. This will let people kick the tires on Kubeflow on their laptops, without having to have a beast of a laptop.

### CODE REVIEW

File: microk8s-resources/actions/enable.kubeflow.sh [hunk 1]
```diff
@@ -106,7 +106,7 @@ def get_hostname():
 
 def main():
     password = os.environ.get("KUBEFLOW_AUTH_PASSWORD") or get_random_pass()
-    bundle = os.environ.get("KUBEFLOW_BUNDLE") or "cs:kubeflow-195"
+    bundle = os.environ.get("KUBEFLOW_BUNDLE") or "cs:kubeflow-206"
```
Comment (ktsakalozos) [inline microk8s-resources/actions/enable.kubeflow.sh:109]: Could we use the syntax we have for passing arguments in addons (`microk8s enable kubeflow:bundle=lite`)?

Arguments (eg "bundle=lite") are passed as the first argument on the executable script [1].

[1] https://github.com/ubuntu/microk8s/blob/master/microk8s-resources/wrappers/microk8s-enable.wrapper#L39

**Comment by ktsakalozos** [comment]
Very happy with this work. It means we can now turn on the kubeflow test we have disabled.

We will also need to update the docs.

File: microk8s-resources/actions/enable.kubeflow.sh (hunk 1 above)
Comment (knkski) [reply microk8s-resources/actions/enable.kubeflow.sh:109]: Given that the rest of the arguments for kubeflow are env vars, can we merge as-is, and open up a new PR that deprecates the env var arguments and switches to the string args?

Comment (ktsakalozos) [reply microk8s-resources/actions/enable.kubeflow.sh:109]: Sounds good. Lets work this on a new PR.

**Pull request merged**